### ISSUE TITLE
How do write in Velocity, the Widget Text with overflow

### ISSUE THREAD

**Issue by GustavoLofrano** [iampawan/VelocityX#31]
How do I write in Velocity, the Widget Text with overflow,

```
Text(
          this.text,
          style: TextStyle(
            fontSize: 14,
          ),
          overflow: TextOverflow.ellipsis,
          textAlign: TextAlign.center,
        ),
      )
```

**Comment by iampawan** [comment]
It will be added in the next version. Great feedback

**Comment by MahdiPishguy** [comment]
@GustavoLofrano @iampawan 

see [this link](https://github.com/iampawan/VelocityX/pull/32/commits/d4cd2be8d9a3d7baad9011976a3f065dcb18fb8a), is it enough

**Comment by GustavoLofrano** [comment]
@MahdiPishguy Wow, I just mentioned the overflow, but you did practically add everything from Text. If version 0.4.2 is updated with this, it will be Excellent.

@iampawan

**Comment by iampawan** [comment]
New version releasing this weekend probably

**Comment by iampawan** [comment]
Fixed in 5dc84210636d581125de246d235698a7ae5464b9
